provisioner/pkg/deleter/jobcontroller.go

```diff
@@ -290,7 +289,21 @@ func NewCleanupJob(pv *apiv1.PersistentVolume, volMode apiv1.PersistentVolumeMod
 		Name:      mountName,
 		MountPath: config.MountDir},
 	}
-
+	if volMode == apiv1.PersistentVolumeBlock {
+		// We need to mount /dev into clean job for block volume.
+		volumes = append(volumes, apiv1.Volume{
+			Name: "dev",
+			VolumeSource: apiv1.VolumeSource{
+				HostPath: &apiv1.HostPathVolumeSource{
+					Path: "/dev",
+				},
+			},
+		})
+		jobContainer.VolumeMounts = append(jobContainer.VolumeMounts, apiv1.VolumeMount{
+			Name:      "dev",
+			MountPath: "/dev",
```

Comment:
I'm assuming this doesn't have any issues if we override the container's /dev?

Reply:
No, to discover and cleanup we need to mount host `/dev` into provisioner and cleanup containers.
Potential issue is in [certain docker setup](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/issues/50) program cannot start when host `/dev` is mounted into container, but provisioner cannot too.

Reply:
Can you add a comment with a link to the issue?

I assume mounting /dev somewhere else won't work because of symlink resolution issues?

Reply:
Updated.

> I assume mounting /dev somewhere else won't work because of symlink resolution issues?
Yes